### PR TITLE
Katello 4.14 release notes

### PR DESCRIPTION
**Pull request by chris1984**
#### What changes are you introducing?

Katello 4.14 Release notes, contibutors, headline features and upgrade warnings

#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)

https://community.theforeman.org/t/katello-4-14-rc1-release-procedure/39182

#### Checklists

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: guides/doc-Release_Notes/master.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/master.adoc:5]: Please insert this at the right place. There's an explicit instruction for this in https://community.theforeman.org/t/katello-4-14-rc1-release-procedure/39182 that does exactly that:
> `sed '/x.y.z releases here/a include::topics/katello-4.14.0.adoc[leveloffset=+1]' master.adoc`

File: guides/doc-Release_Notes/topics/katello-4.14.0.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello-4.14.0.adoc:16]: Not blocking, but ideally these would be categorized in Redmine.

File: guides/doc-Release_Notes/topics/katello-4.14.0.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello-4.14.0.adoc:59]: I think this was reverted and shouldn't be in the changelog. I reopened the issue.

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:9]: We put sentences on their own lines. Quoting https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md#pull-request-checklist
> I put each sentence to its own line.

See https://sembr.org/ for more.

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:16]: Should this be a bullet list?
```suggestion
* Manage content
* Packages (install / remove / upgrade)
* Errata
* Content source
* Content view environments - change the content view and lifecycle environment of hosts
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:7]:
```suggestion
=== New All Hosts index page improvements
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:22]:
```suggestion
=== Multiple Content Views
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:24]: This feels like a header.

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:47]: Please give this its own header

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:45]: https://github.com/theforeman/foreman-documentation/pull/3180 should also be included. If @ehelms doesn't finish it then you should take over the effort.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (chris1984) [reply guides/doc-Release_Notes/topics/katello.adoc:45]: @ehelms it looks like they want you to rebase on the 3.12 branch. https://github.com/theforeman/foreman-documentation/pull/3180#pullrequestreview-2254314673

Did you want me to just take your note and add this to mine?

**Review by chris1984 — COMMENTED** [review]
@ekohl updated all the items, even categorized the Redmine issues. The only thing I have not touched yet is the release note from Eric, I left him a comment on the thread where you pinged him to ask if he wants me to add it here.

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:12]:
```suggestion
*Host bulk actions* - You can now select one or more hosts and click the vertical ellipsis menu to perform the following actions:

```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:23]:
```suggestion
Any changes made in either the new experimental *All Hosts* page and the legacy *All Hosts* page reflect on each other.
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:27]:
```suggestion
You can now assign multiple content view environments to a host through subscription-manager or Hammer. A multi-environment host will have access to enabled repositories from all of its assigned content view environments. This strategy may be more convenient than composite content views for instances where frequent publishes are required.
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:29]:
```suggestion
Through hammer, pass the `content-view-environments` param to `hammer host update` in a comma-separated list in the format `<lifecycle_environment_label>/<content_view_label>` . Ex `hammer host update --id xxx --content-view-environments lce1/cv1,lce2/cv2`
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [inline guides/doc-Release_Notes/topics/katello.adoc:43]:
```suggestion
Assigning or registering a host to multiple environments requires the *Allow multiple content views* setting to be on. 
```

**Comment by chris1984** [comment]
@ekohl @bangelic how does this look? I got all the issues addressed.

File: guides/doc-Release_Notes/master.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/master.adoc:16]: Looks like we need to enhance the procedure to only show it for Katello
```suggestion
ifdef::katello[]
include::topics/katello-4.14.0.adoc[leveloffset=+1]
endif::[]
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:26]: I think this reads better, but I'm looking at @bangelic for a definitive answer
```suggestion
=== Assign multiple content view environments to hosts
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:35]:
```suggestion
To see the list of possible content view environments, use `subscription-manager environments --list`
To see the currently assigned content view environment(s), use `subscription-manager environments --list-enabled`
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:65]:
```suggestion
Extra Pulpcore indexes to address a https://github.com/pulp/pulpcore/issues/5367[performance issue] will be added during the upgrade.
If you've manually created these indexes, please remove them before proceeding with the upgrade to ensure that the migration runs successfully.
```

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:47]:
```suggestion
You can also assign multiple environments at registration using `subscription-manager register --environments`, but global registration and activation keys are not yet supported.
```

**Review by ekohl — CHANGES_REQUESTED** [review]

File: guides/doc-Release_Notes/topics/katello.adoc
Comment (bangelic) [reply guides/doc-Release_Notes/topics/katello.adoc]: I agree with @ekohl

**Comment by chris1984** [comment]
@ekohl committed all your suggestions and rebased to one commit, when you have time to check again. Hopefully, third time is a charm :)

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
@bangelic please merge if you think this is ready.

**Comment by bangelic** [comment]
@Lennonka or @asteflova can one of you merge this? It got the ACK from @ekohl. Thanks!

**Comment by ekohl** [comment]
Oh yes, I forgot you don't have merge permissions.

**Pull request merged**
